cdap-api/src/main/java/co/cask/cdap/api/customaction/AbstractCustomAction.java

```diff
@@ -58,6 +61,7 @@ protected void setProperties(Map<String, String> properties) {
   }
 
   @Override
+  @TransactionPolicy(TransactionControl.IMPLICIT)
```

Comment:
I see the addition of this annotation across all abstract program classes. Is it necessary or is it just a way to document what's the default transaction policy is?